tests/python/mock_jira.py

```diff
@@ -712,9 +726,11 @@ def search_assignable_users_for_issues(self, name, startAt=0, maxResults=20, *ar
             # Mock Jira REST api bug
             return []
 
-        if startAt == 0:
-            return [User(None, None, JIRA_USER_2)] * maxResults
-        return [User(None, None, JIRA_USER)]
+        # Create a list of users.
+        users = [User(None, None, JIRA_USER_2)] * maxResults + [User(None, None, JIRA_USER)]
```

Comment:
The method should return an empty array when you are out of bound. This caused issue during one of the new tests.